src/main/java/me/jellysquid/mods/sodium/client/render/pipeline/BlockRenderer.java

```diff
@@ -102,7 +102,7 @@ public boolean renderModel(IBlockAccess world, RenderBlocks renderBlocks, Block
 
                 for (ModelQuadFacing facing : ModelQuadFacing.VALUES) {
                     this.random.setSeed(seed);
-                    this.renderQuadList(pos, lighter, buffers, quads, facing, block.getRenderType() == 0);
+                    this.renderQuadList(pos, lighter, buffers, quads, facing, this.useAmbientOcclusion);
```

Comment:
I use `useSodiumLight` later on in `renderQuadList` to disable side checking for quads - that probably needs to be removed now, since it's being enabled for the tesselator too.